.github/workflows/main.yml

```diff
@@ -11,13 +11,35 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Cache cargo
+
+      - name: Detect the installed Rust version
+        id: rustc
+        run: echo "::set-output name=version::$(rustc -V)"
```

Comment:
Was this deprecated as part of https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/ ? Or just `set-env` and set-output is still fine?

Reply:
Only `set-env` was deprecated.